### DISCUSSION TITLE
How to set next-auth behind http proxy agent?

### DISCUSSION THREAD

**Discussion by ZyairYH** [nextauthjs/next-auth#10063] [Help]
1. next-auth: 5.0.0-beta.11
2. nextjs: 14.1.1-canary.59

![1708250183324](https://github.com/nextauthjs/next-auth/assets/19263658/12dc6113-f56c-4468-b71d-0dcc0f0d81b5)

i see next-auth use global fetch(undici? i guess) to make request
is there a config or hack way to make next-auth fetch behind a proxy agent like http://127.0.0.1:7890

**Comment by aakash14goplani** [comment]
@ZyairYH - were you able to figure out solution?
